### PR TITLE
Examples-inspired integtests

### PR DESCRIPTION
**Pull request by dk1844**
The `SampleMeasurements{1, 2, 3}` served a basis for this integration tests.
 - the original code has been extended to perform additional checks on the resulting `_INFO` file content to make the test complete.
 - the input files (`atum/src/test/resources/data/input/{wikidata.csv, wikidata.csv.info}`) are directly copied from the `examples` module
 - The examples also contain ControlMeasure-creating example code, this is IMHO covered sufficiently in the `ControlMeasureBuilderTest`
   - the check has been extended to cover the recently-introduced `cmb.withSoftware(...)`, too.

I haven't created a specific integration profile at this point, because the breadth of the integration code is quite small. Should it grow significantly, we can always proceed with this further.

### CODE REVIEW

File: build-all.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 # ------------------------------------------------------------------------
-# Copyright 2020 ABSA Group Limited
```
Comment (dk1844) [inline build-all.sh:3]: This change is just to have a conforming header in all files.

File: atum/src/test/scala/za/co/absa/atum/integration/SampleMeasurementsIntegrationSuite.scala [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+      // showing that "total_response_size" got renamed to "trs"
+      checkpoint1.controls.map(_.controlCol) should contain("total_response_size")
+      checkpoint2.controls.map(_.controlCol) should (contain("trs") and not contain ("total_response_size"))
+    }
+
+    spark.disableControlMeasuresTracking()
+  }
+}
+
+object SampleMeasurementsIntegrationSuite {
```
Comment (Zejnilovic) [inline atum/src/test/scala/za/co/absa/atum/integration/SampleMeasurementsIntegrationSuite.scala:127]: I am unsure if we have an agreement on this or not, but I would rather see this at the top

**Review by Zejnilovic — APPROVED** [review]
I like it

File: atum/src/test/scala/za/co/absa/atum/integration/SampleMeasurementsIntegrationSuite.scala (hunk 2 above)
Comment (dk1844) [reply atum/src/test/scala/za/co/absa/atum/integration/SampleMeasurementsIntegrationSuite.scala:127]: moved in https://github.com/AbsaOSS/atum/pull/106

**Pull request merged**